app/interactors/users/checkins/import_checkins.rb

```diff
@@ -14,6 +15,10 @@ def call
 
     private
 
+    def creat_activity
+      device.create_activity :import, owner: device.user, parameters: { count: CSV.read(path).length }
```

Comment:
Looks like all these activity extra actions are very similar. What if we move this logic into some activity actor (who's action is 'create_activity' by default) and will pass `entity`, 'activity', `owner` and `params` there so it could be something like:
`entity.public_send :action, activity, owner: owner, parameters: params`

Does it make sense?